uv.lock

```diff
@@ -1,10 +1,6 @@
 version = 1
```

Comment:
Hey @felix314159, as we discussed, can you please revert the updates to all packages and only modify the required packages. 

Instructions available here:
https://eest.ethereum.org/main/dev/deps_and_packaging/